apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -235,7 +235,7 @@ func (m *ModelManagerAPI) CreateModel(ctx context.Context, args params.ModelCrea
 		creationArgs.Credential = credential.KeyFromTag(cloudCredentialTag)
 	}
 
-	user, err := m.accessService.GetUserByName(ctx, user.NameFromTag(ownerTag))
+	userUUID, err := m.accessService.GetUserUUIDByName(ctx, user.NameFromTag(ownerTag))
```

Comment:
This will be deleted when the model namespace replaces owner - PR ready to propose after this lands.
I wonder if we want to avoid changing the service and just make the model->user FK optional and use the "admin" user for the model owner for now. Then we avoid introducing a whole new method only to then delete it next PR.

Reply:
I don't see the point at the moment. It was done to address a need at the time. Would rather wait to see the PR that is incoming. Ideally we still want to be turning natural keys into uuids at the facade layer and working on this value for the rest of the stack.

I don't see how that need will go away even with a new design ?

Reply:
There won't be a model to user relation at all. Models won't have owners.